README.md

```diff
@@ -1,15 +1,23 @@
 # Social-Indicators-Visualization
 
-To get the code and data, use these commands in a terminal:
+To get the code, use these commands in a terminal:
 
+```
```

Comment:
When pasting code in a markdown document, you can use the triple backtick to format it: ```